### PR TITLE
Add techmmunity eslint config

### PR DESCRIPTION
**Pull request by aleessandrohr**
## What this PR introduces?

Add Techmmunity Style Guide

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] I followed GitFlow pattern to create the branch
- [x] My code produces no warnings or errors in the console
- [x] Tests for the changes have been added / updated
- [x] The changes made are accessible
- [x] The changes made are responsive

## PR Type

What kind of change does this PR introduce?

```
[ ] Hotfix
[ ] Bugfix
[ ] Feature
[ ] Documentation update
[x] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] CI/CD related changes
```

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,23 +22,28 @@
...
 		"@types/react-dom": "16.8.4",
 		"@types/styled-components": "5.0.1",
-		"@typescript-eslint/eslint-plugin": "^4.21.0",
-		"@typescript-eslint/parser": "^4.21.0",
+		"@typescript-eslint/eslint-plugin": "^4.26.0",
+		"@typescript-eslint/parser": "^4.26.0",
 		"babel-plugin-styled-components": "^1.10.0",
-		"eslint": "^7.23.0",
+		"eslint": "^7.27.0",
 		"eslint-config-prettier": "^8.1.0",
```
Comment (henriqueleite42) [inline package.json:34]: Essa config aqui eu tenho quase certeza que pode ser removida, remover e testar, por favor.

File: src/pages/[language]/index.tsx [hunk 2]
```diff
@@ -21,7 +22,7 @@ export const getStaticPaths: GetStaticPaths = async () => {
 };
 
 export const getStaticProps: GetStaticProps = async ({ params }) => {
-	const language = (params?.language || i18n.defaultLocale) as string;
+	const language = (params?.language ?? i18n.defaultLocale) as string;
```
Comment (henriqueleite42) [inline src/pages/[language]/index.tsx:25]: Reverter alteração

File: src/types/window.d.ts [hunk 3]
```diff
@@ -4,7 +4,9 @@ declare global {
 	}
 }
 
-// If this file has no import/export statements (i.e. is a script)
-// convert it into a module by adding an empty export statement.
+/*
+ * If this file has no import/export statements (i.e. is a script)
+ * Convert it into a module by adding an empty export statement.
+ */
 // eslint-disable-next-line prettier/prettier
```
Comment (henriqueleite42) [inline src/types/window.d.ts:11]: Remover essa linha (Se o eslint reclamar é por cause de algum formatador que está dando conflito com o eslint)

File: src/utils/gtm.ts [hunk 4]
```diff
@@ -3,7 +3,7 @@ interface PageEventProps {
 	page: string;
 }
 
-export const GTMPageView = (url: string) => {
+export const gtmPageView = (url: string) => {
```
Comment (henriqueleite42) [inline src/utils/gtm.ts:6]: Alterar nome para `setGtmPageView`

**Review by henriqueleite42 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (alessandrohenriqueramos) [reply package.json:34]: precisa do pacote

**Review by henriqueleite42 — APPROVED** [review]

**Pull request merged**